internal/providers/terraform/aws/cloudformation_stack.go

```diff
@@ -0,0 +1,80 @@
+package aws
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetCloudFormationStackRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_cloudformation_stack",
+		RFunc: NewCloudFormationStack,
+	}
+}
+
+func NewCloudFormationStack(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	if d.Get("template_body").Type != gjson.Null && checkAWS(d) || checkAlexa(d) || checkCustom(d) {
```

Comment:
I think it would be better if you separate `and` logic by brackets.
```suggestion
	if d.Get("template_body").Type != gjson.Null && (checkAWS(d) || checkAlexa(d) || checkCustom(d)) {
```